src/DataObjects/WorkflowDefinition.php

```diff
@@ -482,12 +482,7 @@ public function incomingTitle()
      */
     public function canWorkflowPublish($member, $target)
     {
-        $publish = $this->extendedCan('canWorkflowPublish', $member, $target);
-
-        if (is_null($publish)) {
-            return false;
-        }
-        return $publish;
+        return $this->extendedCan('canWorkflowPublish', $member, $target);
```

Comment:
I think we should still fail to `ADMIN` check in case `$publish` is null (rather than return false).

Reply:
See reasoning at https://github.com/symbiote/silverstripe-advancedworkflow/issues/333#issuecomment-362972994